src/cc/arduino/plugins/wifi101/flashers/cli/CLIFlasher.java

```diff
@@ -50,7 +50,7 @@ public CLIFlasher() throws Exception {
 		try {
 			String jarPath = CLIFlasher.class.getProtectionDomain().getCodeSource().getLocation().toURI().getPath();
 			File jarFolder = new File(jarPath).getParentFile();
-			String toolName = "winc1500-uploader";
+			String toolName ="winc1500-uploader";
```

Comment:
Remember please match the existing code style, and don't introduce changes like this!